src/agent/onefuzz/src/input_tester.rs

```diff
@@ -228,11 +256,14 @@ impl<'a> Tester<'a> {
                     CRASH_SITE_UNAVAILABLE.to_owned()
                 };
 
-                Some(Crash {
-                    call_stack,
-                    crash_type,
-                    crash_site,
-                })
+                let text = "".to_string();
```

Comment:
Hmm, in `CrashLog`, should `text` maybe be wrapped in an `Option<_>`?

Reply:
yes, eventually.  it maps to asan_log via a long chain of events, which isn't optional at the moment.  My thought was that should be a follow-on PR.

Reply:
SGTM!